__test__/Radio.test.js

```diff
@@ -0,0 +1,135 @@
+const isChecked = (elementHandle) =>
+  page.evaluate((element) => {
+    const innerCheckbox = element.shadowRoot.querySelector("input");
+    return innerCheckbox.checked;
+  }, elementHandle);
+
+describe("Radio", () => {
+  const getPageURL = (size, label, labelPosition, checked, state) =>
+    `http://localhost:6006/iframe.html?viewMode=story&id=components-radio--small&args=size:${size};label:${label};labelPosition:${labelPosition};checked:!${checked};${state == "disabled" ? "disabled:!true" : ""}`;
+  // vision test
+  describe.each(["small", "large"])("%s", (size) => {
+    describe.each(["left", "right"])("%s", (labelPosition) => {
+      describe.each(["checked", "unchecked"])("%s", (checked) => {
+        describe.each(["enabled", "disabled"])("%s", (state) => {
+          const baseURL = getPageURL(
+            size,
+            "test label",
+            labelPosition,
+            checked === "checked",
+            state
+          );
+
+          it("enable", async () => {
```

Comment:
Not consistent with disabled.

```suggestion
          it("base", async () => {
```